### PR TITLE
Post/exit calls

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @lubc @ercohen14 @mauro-oto, 

I just wrote an article about our exit calls. Please check it out and let me know.

Thanks!

### CODE REVIEW

File: _posts/2018-09-12-fastruby-io-exit-calls.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Conclusion
+
+Exit calls have been a great way to learn from our relationships with clients.
+We have written a few case studies based on these exit calls. For example:
+[Upgrading a Huge Monolith from Rails 4.0 to 5.1](https://www.ombulabs.com/blog/rails/upgrades/case-study/upgrading-a-monolith.html)
+
+Our secret sauce for our productized [Rails upgrade service](https://fastruby.io) ([https://fastruby.io](https://fastruby.io))
+is constantly changing, getting better with every project we ship. Exit calls are
+the best way to keep getting better.
```
Comment (lubc) [inline _posts/2018-09-12-fastruby-io-exit-calls.markdown]: @etagwerker Just to not repeat `getting better`, maybe something like `Exit calls are the best way to keep improving as a company` sound better?

File: _posts/2018-09-12-fastruby-io-exit-calls.markdown [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title:  "FastRuby.io: Exit Calls"
+date: 2018-09-12 11:10:00
+categories: ["continuous-learning"]
+author: "etagwerker"
+---
+
+At Ombu Labs we believe that if we are not learning from your mistakes we are
```
Comment (mauro-oto) [inline _posts/2018-09-12-fastruby-io-exit-calls.markdown]: I think you meant to say "if we are not learning from our mistakes" instead.

File: _posts/2018-09-12-fastruby-io-exit-calls.markdown [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Here are some of the questions that we like to ask:
+
+- How was it doing business with us?
+
+- Is there anything we could have done differently?
+
+- What did you _like_ about our service?
+
+- What did you _hate_ about our service?
+
```
Comment (ercohen14) [inline _posts/2018-09-12-fastruby-io-exit-calls.markdown]: Maybe this could be "what did you dislike" instead of hate.

**Comment by lubc** [comment]
@etagwerker Cool! Looks good to me 👍

**Pull request merged**